themes/src/main/resources/theme/base/login/login-otp.ftl

```diff
@@ -9,13 +9,13 @@
                         <div class="${properties.kcFormGroupClass!}">
                             <div class="${properties.kcInputWrapperClass!}">
                                 <#list otpLogin.userOtpCredentials as otpCredential>
-                                    <div class="${properties.kcSelectOTPListClass!}">
+                                    <div class="${properties.kcLoginOTPListClass!}" tabindex="${otpCredential?index}">
                                     <input type="hidden" value="${otpCredential.id}">
-                                        <div class="${properties.kcSelectOTPListItemClass!}">
-                                            <span class="${properties.kcAuthenticatorOtpCircleClass!}"></span>
-                                            <h2 class="${properties.kcSelectOTPItemHeadingClass!}">
-                                                ${otpCredential.userLabel}
-                                            </h2>
+                                        <div class="${properties.kcLoginOTPListItemHeaderClass!}">
+                                            <div class="${properties.kcLoginOTPListItemIconClass!}">
+                                              <i class="fa fa-mobile" aria-hidden="true"></i>
```

Comment:
I think, it'd be better use property for the icon instead of hardcoded value. Then, customer can easily change the icon in `theme.properties`. WDYT @Pepo48 ?

Reply:
@mabartos (cc: @vmuzikar ) fixed